delivery_package_gs1_barcode_report/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Print a label for outgoing stock packages, including a GS1 barcode.
```

Comment:
Actually it's multiple GS1 barcodes - describe which ones are included and what they are for.